protocol/x/prices/keeper/market_param.go

```diff
@@ -14,58 +14,59 @@ import (
 	"github.com/dydxprotocol/v4-chain/protocol/x/prices/types"
 )
 
-// newMarketParamStore creates a new prefix store for MarketParams.
-func (k Keeper) newMarketParamStore(ctx sdk.Context) prefix.Store {
+// getMarketParamStore returns a prefix store for MarketParams.
+func (k Keeper) getMarketParamStore(ctx sdk.Context) prefix.Store {
 	return prefix.NewStore(ctx.KVStore(k.storeKey), []byte(types.MarketParamKeyPrefix))
 }
 
 // ModifyMarketParam modifies an existing market param in the store.
 func (k Keeper) ModifyMarketParam(
 	ctx sdk.Context,
-	marketParam types.MarketParam,
+	newMarketParam types.MarketParam,
```

Comment:
I like this change to clarify that this variable is the updated version of the param. WDYT about `updatedMarketParam` to be more closely in line with the method name / function?